tests/platform/mellanox/check_sysfs.py

```diff
@@ -122,6 +122,23 @@ def check_sysfs(dut):
     psu_count = SWITCH_MODELS[dut_hwsku]["psus"]["number"]
     for psu_id in range(1, psu_count + 1):
         if SWITCH_MODELS[dut_hwsku]["psus"]["hot_swappable"]:
+
+            # if the PSU if poweroff, all thermal related sensors of the PSU is removed.
```

Comment:
typo. suggest to correct it and consider the following
"If the PSU is poweroff, all PSU thermal related sensors are not available."

Reply:
Fixed.